js/editor-cm.js

```diff
@@ -1,5 +1,99 @@
 var EditSession = CodeMirror.Doc;
 
+// The following code was taken from chrome dev tools to help improve codemirror
```

Comment:
'the following code' implies the rest of the file - change to 'the devToolsAccessibleTextArea class'?

Reply:
will update!